lms/djangoapps/bulk_email/models.py

```diff
@@ -107,7 +107,7 @@ def get_users(self, course_id, user_id=None):
         staff_qset = CourseStaffRole(course_id).users_with_role()
         instructor_qset = CourseInstructorRole(course_id).users_with_role()
         staff_instructor_qset = (staff_qset | instructor_qset)
-        enrollment_qset = User.objects.filter(
+        enrollment_query = models.Q(
```

Comment:
Its my belief that `Q` objects are generally used for more complex lookups like at time where we need 'OR'ing of query params, so if we can achieve our goal (dynamic query building) with `kwargs`, it would result into an easy to understand/maintain code.
I am good with use of `Q` objects here also with abstraction achieved.